### DISCUSSION TITLE
[DRM] Draw Bitmap on initialization

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#15122] [Q&A]
Hi,
I saw that it is possible to set the initial color, I would like to ask if it is also possible to render a static image.

https://github.com/AvaloniaUI/Avalonia/blob/029daf79c97e7fed493e00b4f562ac0f43bfe916/src/Linux/Avalonia.LinuxFramebuffer/DrmOutputOptions.cs#L26

**Comment by maxkatz6** [comment]
You probably can use Compositor.TryGetDefaultCompositor() to draw something earlier, but I am not sure about exact usage.
https://github.com/AvaloniaUI/Avalonia/pull/14651

**Comment by kekekeks** [comment]
You can probably call drmOutput.CreateGlRenderTarget and draw something manually, which will work for at least 11.0.x/11.1.x.

There are plans for proper splash support, but no ETA

**Comment by workgroupengineering** [comment]
Thanks to both of you for the suggestions.

> There are plans for proper splash support, but no ETA
If you have any specs, I can try to implement them when I have some free time.
